client/src/components/MoreHomes.jsx

```diff
@@ -35,10 +46,22 @@ class MoreHomes extends React.Component {
   render() {
     return (
       <div className="module">
-        <ListOfHomes data={this.state.data} />
+        <div className="moduleTitle">
+          <span>More homes you may like</span>
+        </div>
+        <ListOfHomes data={this.state.data}
+          beginIndex={this.state.imgBegin}
+          endIndex={this.state.imgEnd}
+          height={this.state.height}
+          width={this.state.width}
+        />
       </div>
     )
   }
+
+  componentWillUnmount() {
+    window.removeEventListener("resize", this.handleResize);
```

Comment:
Good job cleaning up after yourself.

Reply:
But this should go above the render method.

Reply:
Done